Networking/Networking/Model/Receipt.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public struct Receipt: Decodable {
+    public let receiptURL: String
```

Comment:
I'm 50/50 about renaming this parameter from `receiptURL` to `receiptURLString` or similar. I went ahead with `receiptURL` for consistency with the API, but I can see the case for being consistent with the data type on the client instead.

Reply:
I suppose the client will need to create `URL` object from it? If we always need to create `URL` on the client maybe we can do it when initializing `Receipt`. Or we have `receiptURLString` and have a computed property `var receiptURL: URL?`?

Reply:
Yes, at some point we'll create the URL object from it, but from my exploration we might not need it until the last moment in the presentation layer when we're actually invoking the WKWebView initializer with an `URL(string: <url-goes-here>)` .

Since we also won't do any validation regarding the contents of the URL through the business layer,  the rest of the app shouldn't care if this is a valid URL or not, the web view will either return a receipt or a 404/other messages when needed, so I think I'm happy to keep it as a simple `String` while the data flows through the app.